### ISSUE TITLE
modal basestyle overriding is not working

### ISSUE THREAD

**Issue by hoondeveloper** [GeekyAnts/NativeBase#5084]
### Description

modal basestyle overriding is not working

### CodeSandbox/Snack link

none

### Steps to reproduce

```ts
const THEME = extendTheme({
  components: {
    Modal: {
      baseStyle: {
        _backdropFade: {
          exitDuration: 10,
          entryDuration: 10,
        },
      },
    },
});
```

modal _backdropFade overriding is not working

### NativeBase Version

3.4.6

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [X] iOS
- [ ] Next

**Comment by Krithikj20** [comment]
Thank you @hoondeveloper for reporting. We will look into it.

**Comment by Viraj-10** [comment]
This issue has been fixed in `v3.4.8`.
